### ISSUE TITLE
vertical gap in image

### ISSUE THREAD

**Issue by jeffyzxc** [clearbold/html-email-resources#1]
Hello

Have you solved your issue in your post here ? https://litmus.com/community/discussions/5959-windows-10-mail-gap-next-to-image

Im encountering the same issue hope you can help me. Saw your github account in litmus haha

TIA 😄

**Comment by heymarkreeves** [comment]
@jeffyzxc NO! :) Every time I test a new version of these emails, it's still there.

**Comment by jeffyzxc** [comment]
Haayss hahaha im going crazy solving that vertical gap.
